package.json

```diff
@@ -38,7 +38,7 @@
     "babel-preset-react": "^6.16.0",
     "babel-preset-stage-0": "^6.16.0",
     "enzyme": "^2.6.0",
-    "jest-cli": "^0.9.2",
+    "jest": "^21.2.1",
```

Comment:
Cpr-mask hasn't changed in a long time, something changed in the version of jest and had thing broken. I upgraded jest and the tests now work.